client/app.js

```diff
@@ -0,0 +1,68 @@
+const form = document.querySelector("#new-post-form");
+const btn = document.querySelector("#msg-btn");
+const newPosts = document.querySelector("show-posts");
+
+form.addEventListener("submit", submitPost);
+btn.addEventListener("click", getMessage);
+
+function getAllPosts() {
+  fetch("http://localhost:3000/posts")
+    .then((r) => r.json())
+    .then(appendPosts)
+    .catch(console.warn);
+}
```

Comment:
Think you forgot to require your posts router in your server file - and in your router file the path for the require of the anonymous model is slightly misspelled soI think this might be the main reason for the front/backend disconnect. In a nutshell the "/posts" endpoint doesn't exist at the moment